Makefile

```diff
@@ -100,9 +101,9 @@ ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: manifests generate fmt vet envtest ## Run tests.
 ifdef TEST
 	@echo Running test $(TEST)
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./api/... ./pkg/... ./controllers/... -run="$(TEST)" -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test $(GO_FLAGS) ./api/... ./pkg/... ./controllers/... -run="$(TEST)" -coverprofile cover.out
```

Comment:
What would be a typical usage of the `GO_FLAGS` variable? Shouldn't we add it to the `e2e-test` target too?

Reply:
Such as ``-v``

Reply:
In #102 I added a `TEST_FLAGS` for suppressing the noisy output from kustomize and kubectl. See [here](https://github.com/k8ssandra/k8ssandra-operator/pull/102/commits/632d112951c8c08db39f9d00173543253ce7ca7d). I chose `TEST_FLAGS` vs `GO_FLAGS` it is intended for the test itself.

Reply:
That's TEST_ARGS ;). The GO_FLAGS was used in the k8ssandra, that's why I took the name.